pkg/driver/driver_openstack.go

```diff
@@ -285,21 +334,28 @@ func (d *OpenStackDriver) Delete(machineID string) error {
 		return err
 	}
 
-	result := servers.Delete(client, instanceID)
-	if result.Err == nil {
-		// waiting for the machine to be deleted to release consumed quota resources, 5 minutes should be enough
-		err = waitForStatus(client, machineID, nil, []string{"DELETED", "SOFT_DELETED"}, 300)
-		if err != nil {
-			return fmt.Errorf("error waiting for the %q server to be deleted: %s", machineID, err)
-		}
-		metrics.APIRequestCount.With(prometheus.Labels{"provider": "openstack", "service": "nova"}).Inc()
-		klog.V(3).Infof("Deleted machine with ID: %s", machineID)
-	} else {
+	err = servers.Delete(client, instanceID).ExtractErr()
+	if err != nil && isNotFoundError(err) == false {
```

Comment:
just a suggestion for consistency `if err != nil && !isNotFoundError(err)`

Reply:
Corrected in both places.